dashboard/src/shared/types.ts

```diff
@@ -187,15 +184,23 @@ export interface IDeploymentStatus {
   availableReplicas: number;
 }
 
+export interface IReleaseWithUpdateInfo extends hapi.release.Release {
+  updateInfo?: IChartUpdateInfo;
+}
+
+export interface IAppOverviewWithUpdateInfo extends IAppOverview {
```

Comment:
as above, I don't think this is necessary over just having the property as an optional one in `IAppOverview`.